code/modules/mob/living/simple_animal/friendly/raven.dm

```diff
@@ -0,0 +1,59 @@
+
+///the perfectly normal raven.
+/mob/living/simple_animal/pet/raven
+	name = "raven"
+	desc = "Quoth the raven nevermore! Just now in space!"
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "raven"
+	icon_living = "raven"
+	icon_dead = "raven_dead"
+	speak_emote = list("caws")
+	health = 7
+	maxHealth = 7
+	attacktext = "bites"
+	melee_damage_lower = 2
+	melee_damage_upper = 2
```

Comment:
```suggestion
	melee_damage_upper = 2
	turns_per_move = 5
```
While testing the raven did move way too often, setting the turns_per_move to 5 (same as parrots) looks like the best option.